CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.14.0] - 2017-01-18
```

Comment:
what ever goes first, need to update it too . 

ref: https://github.com/edx/edx-enterprise/pull/33/files#diff-de0a9b67ffe2874a076b5fdb15bad484R17